docs/root/install/getting_started.rst

```diff
@@ -0,0 +1,30 @@
+.. _install_getting_started:
+
+Getting Started
+===============
+
+This section features sample configurations and guides to quickly getting started with Envoy.
+
+Sandboxes
+---------
+
+We've created a number of sandboxes using Docker Compose that set up different environments to test out Envoy's features and show samples configurations. As we gauge people's interests we will add more sandboxes demonstrating
```

Comment:
Nit: s/samples configurations/sample configurations/
Nit: s/people's/peoples'/

Reply:
nit: please also try to wrap around 100 cols for docs

Reply:
Fixed the typos. Will switch back to Emacs for working on the Envoy docs just for Matt ;->.